client/src/components/QuestionsAnswers/QuestionForm.jsx

```diff
@@ -1,10 +1,9 @@
 import React, { useState, useEffect } from 'react';
+import { Modal, Button, Form, InputGroup } from 'react-bootstrap';
 import axios from 'axios';
```

Comment:
It might be hidden somewhere, but are you doing any axios requests in this component?

Reply:
yup, my QuestionForm takes the user-inputted answers in the form and then sends a POST request to the API with the question details.